sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -537,8 +554,14 @@ def refresh(self, **kwargs) -> None:
                 self._feature_flag_enabled,
                 self._feature_filter_usage,
                 self._uses_key_vault,
+                self._uses_load_balancing,
+                has_failed,
             )
```

Comment:
Do we need to move this inside the `while` loop?

Reply:
This is within the loop? You might need to refresh I did change this recently.